core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/DetailedSample.java

```diff
@@ -117,4 +118,8 @@
    */
   Optional<DetailedSample> getEffectiveGroupIdSample();
 
+  Date getCreationDate();
```

Comment:
Please Javadoc this and the setter to clarify that this is the user-specified creation date